src/app/cli/src/coda.ml

```diff
@@ -493,7 +493,12 @@ let setup_daemon logger =
                    signal $signal"
                   ~metadata:
                     ( ("signal", `String (Signal.to_string signal))
-                    :: child_pid_metadata )
+                    :: child_pid_metadata ) ;
+                Option.value_map
+                  (Child_processes.Termination.get_signal_cause_opt signal)
+                  ~default:() ~f:(fun cause ->
+                    [%log info] "Possible reason signal caused termination"
```

Comment:
Should this be at level `error` instead? Also, `:` after `possible reason`

Reply:
The other log messages here about termination are at `info` level. Should that change?

I meant the text as, "the possible reason that the signal caused termination".

Reply:
`Possible reason for signal: $cause` may be clearer, and will show the cause it the pretty logs